src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -152,5 +153,73 @@ public static void SyncRoot_Get_Success()
             CookieCollection cc = new CookieCollection();
             Assert.Equal(cc, cc.SyncRoot);
         }
+
+        [Fact]
+        public static void CopyTo_Array_Success()
+        {
+            CookieCollection cc = CreateCookieCollection1();
+            Array c = new object[cc.Count];
+            cc.CopyTo(c, 0);
+            EnsureEqual(cc, c);
+        }
+
+        [Fact]
+        public static void CopyTo_CookieArray_Success()
+        {
+            CookieCollection cc = CreateCookieCollection1();
+            Cookie[] c = new Cookie[cc.Count];
+            cc.CopyTo(c, 0);
```

Comment:
the Pr mentioned has been merged. I'll update the tests to make sure everything works correctly